content/ops/managing-users.md

```diff
@@ -5,6 +5,12 @@ menu:
 title: Managing Users
 ---
 
+## Note for GovCloud environment
```

Comment:
How do readers/users know whether they're in the GovCloud environment? I'd like us to add a little context here explaining how to tell, so readers know for 100% certain whether this note applies to them.